python/sandbox/neurospace/test_nspace.metta

```diff
@@ -13,6 +13,12 @@
 
 ; !(match (my-nspace) (What is the user name $x) $x)
 ; !(match (my-nspace) (How old $x is William) $x)
-!(match (my-nspace) (How old ($x) is Mike) $x)
+; !(match (my-nspace) (How old ($x) is Mike) $x)
 ; !(match (my-nspace) (What $x is father name) $x)
 ; !(match (my-nspace) (What $x is Mike age) $x)
+
+; (match (my-nspace) (What are the names $x of William brothers?) $x)
```

Comment:
Given this is back-ending to an LLM, do you think traditional English grammar is better?  i.e. "What are the names, $x, of William's brothers?" or "What are the names, $x, of the brother's of William?"

Or is a more sparse and regular syntax better?  I am sure the LLM would be able to handle either.

Reply:
Unfortunately `'` doesn't work (in "brother's") because these are not strings. Surrounding `$x` with `,` sometimes work, but gpt seems to be sensitive to the syntax in this case. It can possibly be made slightly more readable in some cases, but the syntax will be less uniform. Since it is just a sandbox example, I'd keep it as is.